src/checkbox/checkbox.js

```diff
@@ -15,255 +15,148 @@
  * limitations under the License.
  */
 
-(function() {
-  'use strict';
+// Numeric constants used in this component.
+const numbers_ = Object.freeze({
+  TINY_TIMEOUT: 0.001
+});
+
+// CSS classes used in this component.
+const classes_ = Object.freeze({
```

Comment:
I considered that, but couldn't find a reasonable way of having a static private property. I'd like it to be static so that instances don't carry copies around, and to be private because it deals with internal implementation details.

Thoughts?